cranelift/codegen/src/isa/riscv64/inst/mod.rs

```diff
@@ -840,7 +849,6 @@ pub fn reg_name(reg: Reg) -> String {
 pub fn vec_reg_name(reg: Reg) -> String {
     match reg.to_real_reg() {
         Some(real) => {
-            assert_eq!(real.class(), RegClass::Float);
```

Comment:
This assert triggers since we are now trying to format some integer registers and I didn't change the pretty printer. However this function gets removed in #6366, so I figured it's easier to wait for that to be merged and rebase this.